src/audiounit.cs

```diff
@@ -56,13 +56,12 @@ namespace AudioUnit {
 // 	AUAudioTODO - We need testing for these bindings
 // 	delegate void AUScheduleMidiEventBlock (AUEventSampleTime eventSampleTime, byte cable, nint length, ref byte midiBytes);
 // 	delegate bool AUHostMusicalContextBlock (ref double currentTempo, ref double timeSignatureNumerator, ref nint timeSignatureDenominator, ref double currentBeatPosition, ref nint sampleOffsetToNextBeat, ref double currentMeasureDownbeatPosition);
-#if !XAMCORE_4_0
+#if !NET
 	[Advice ("The signature will change in the future to return a string")]
```

Comment:
Is this the change for the future? If so, do we still need this advice?

Reply:
yep needed for legacy